docs/src/python/polars_arrow.md

```diff
@@ -2,6 +2,9 @@
 
 LanceDB supports [Polars](https://github.com/pola-rs/polars), a blazingly fast DataFrame library for Python written in Rust. Just like in Pandas, the Polars integration is enabled by PyArrow under the hood. A deeper integration between Lance Tables and Polars DataFrames is in progress, but at the moment, you can read a Polars DataFrame into LanceDB and output the search results from a query to a Polars DataFrame.
 
+!!! note
+    This example uses synchronous APIs as asynchronous `to_polars` support is under development.
+
```

Comment:
Same thing here. We don't plan on implementing async to_polars (Polars itself is entirely sync), so let's not promise this in our docs.